applications/server/controllers/synchronization.py

```diff
@@ -102,12 +106,153 @@ def POST(*args, **vars):
 
     return locals()
 
+"""
+    This method is to receive value data.  It accepts two HTTP methods:
+        GET: returns all the current output entries (for the device id)
```

Comment:
Maybe say whether these things are synched up or down. 
These are synched up, right? 
So why a GET returns them?  Is this done for the web UI?  This is fine, but be sure to coordinate with them on this.